Formula/e/erlang.rb

```diff
@@ -35,11 +34,26 @@ class Erlang < Formula
   depends_on "wxwidgets" # for GUI apps like observer
 
   uses_from_macos "libxslt" => :build
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "coreutils" # sha(1|256)sum used for ex_doc download validation
+  end
+
+  on_linux do
+    depends_on "mesa-glu"
+  end
 
   resource "html" do
-    url "https://github.com/erlang/otp/releases/download/OTP-26.2.5/otp_doc_html_26.2.5.tar.gz"
-    mirror "https://fossies.org/linux/misc/otp_doc_html_26.2.5.tar.gz"
-    sha256 "7c9c99695f8b0218f655ec154954bac1d70e25fe3d54810cf81590e1eb711e47"
+    url "https://github.com/erlang/otp/releases/download/OTP-27.1/otp_doc_html_27.1.tar.gz"
+    mirror "https://fossies.org/linux/misc/otp_doc_html_27.1.tar.gz"
+    sha256 "b0e6d508449eb00f6dd762f5e8ba6eadd0ea3785c79782a3b68c641996d9d168"
+  end
+
+  resource "ex_doc" do
```

Comment:
```suggestion
  # https://github.com/erlang/otp/blob/#{version}/make/ex_doc_link
  resource "ex_doc" do
```
We should add this to make it a bit clearer how to update this resource.